src/main/asciidoc/_chapters/developer.adoc

```diff
@@ -899,6 +910,15 @@ Usage: hbase-vote.sh -s | --source <url> [-k | --key <signature>] [-f | --keys-f
                                 http://www.apache.org/dist/hbase/KEYS
   -o | --output-dir '</path>'   directory which has the stdout and stderr of each verification target
 ----
+** If you see any unit test failures, please call out the solo test result and whether it's part of flaky (nightly) tests dashboard. Link to the dashboard is e.g. https://builds.apache.org/view/H-L/view/HBase/job/HBase-Find-Flaky-Tests/job/<branch-name>/lastSuccessfulBuild/artifact/dashboard.html
```

Comment:
this rendered as a link to an incorrect URL. Can you make it an example of a working link with branch-2.2?

Reply:
fixed, I changed to master branch with a note one should change the test branch accordingly such we don't have to change this ref guide often.